src/include/daos_srv/bio.h

```diff
@@ -1079,4 +1079,57 @@ bool bio_meta_is_empty(struct bio_meta_context *mc);
  */
 int bio_meta_clear_empty(struct bio_meta_context *mc);
 
+/*
+ * Wrapper of ds_notify_ras_event.
+ */
+static inline void
+bio_notify_ras_event(ras_event_t id, char *msg, ras_type_t type, ras_sev_t sev, char *hwid,
```

Comment:
Is there anything about this that's bio specific? I wonder if this would belong somewhere more general, maybe ras.h?

Reply:
These APIs are used for bio and vos module to make vos test workable without full engine mode, i am fine to move it ras.h but there is already ds_xxx apis there, any good name for this case? so i keep it on bio.h for now.

Reply:
yeah, moving this wrapper function to a ras.h looks more reasonable. What about ras_notify_event()?